stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/ChatInitializer.kt

```diff
@@ -12,6 +12,11 @@ import io.getstream.chat.ui.sample.R
 
 class ChatInitializer(private val context: Context) {
 
+    init {
+        // ChatUI must be initialized before any io.getstream.chat UI component is rendered
+        val ui = ChatUI.Builder(context).build()
```

Comment:
This is moving the initialisation of `ChatUI` to `App`. 
I would prefer to create a method for it like `initUIComponent()` (or any other name) and call it in the `App::onCreate` instead of using the constructor of `ChatInitializer`. It would make clearer what it happening and we wouldn't be using a constructor for something it is not supposed to do.

Reply:
Crashes with
```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference
        at android.content.ContextWrapper.getResources(ContextWrapper.java:93)
        at io.noties.markwon.utils.Dip.create(Dip.java:11)
        at io.noties.markwon.core.MarkwonTheme.builderWithDefaults(MarkwonTheme.java:93)
        at io.noties.markwon.MarkwonBuilderImpl.build(MarkwonBuilderImpl.java:98)
        at com.getstream.sdk.chat.ChatMarkdownImpl.<init>(ChatMarkdownImpl.java:24)
        at com.getstream.sdk.chat.ChatUI$Builder.build(ChatUI.kt:93)
        at io.getstream.chat.ui.sample.application.ChatInitializer.<init>(ChatInitializer.kt:17)
        at io.getstream.chat.ui.sample.application.App.<init>(App.kt:14)
        at java.lang.Class.newInstance(Native Method)
```

Reply:
This is now moved to `App::onCreate` so I guess this should solve the issue.

Reply:
Anyway, I wasn't able to reproduce it before, so could you verify the crash doesn't happen now?

Reply:
lemmi check it...

Reply:
I can't reproduce the problem o.O